### DISCUSSION TITLE
Is there any interest in a de-cluttered UI revision?

### DISCUSSION THREAD

**Discussion by jibbel23** [kagemomiji/airsonic-advanced#371] [Ideas]
I know there's a discussion around a new UI framework to replace the old JSP stuff, but just wanted to see if there was any interest in a few revisions to the legacy stack as a stop-gap. I added below improvements to the 11.1.3 stable branch for personal use, but happy to create a branch if others think it'd be useful.

- Improved screen-size responsiveness for better smart TV and desktop UX
- Re-organized header icons and layout
- Removed unused features in left folder view (playlists, artists, singles, etc). Sticky shortcuts.
- Album and Play Queue optimizations to de-clutter UI elements
- Disabled pagination globally

Screenshots: [Example 1](https://i.imgur.com/gQqQWt4.png), [Example 2](https://i.imgur.com/hMJrSO4.png), [Example 3](https://i.imgur.com/CwvbO7C.png)

**Comment by kagemomiji** [comment]
@jibbel23 
Thank you for your suggestions.
I am in favor of the following three improvements:

- Improved screen-size responsiveness for better smart TV and desktop UX
- Re-organized header icons and layout
- Album and Play Queue optimizations to de-clutter UI elements

As for removing unused features in the left folder view (playlists, artists, singles, etc.) and sticky shortcuts, I believe there are users who still utilize these features (for example, I use playlists), so I am on the fence about this.

Regarding disabling pagination, since it's possible to set the value to -1 in Personal Settings to effectively disable it, I don't think there's a need to completely remove this feature.
![image](https://github.com/kagemomiji/airsonic-advanced/assets/5343692/68669b07-5a23-407e-b388-6221ac9eadbd)
